addon/-private/data-view/radar/radar.js

```diff
@@ -255,20 +306,43 @@ export default class Radar {
       _componentPool,
 
       shouldRecycle,
+      renderAll,
+
       _didReset,
+      _started,
 
-      itemContainer,
       _occludedContentBefore,
       _occludedContentAfter,
-      totalItems,
-
-      renderedFirstItemIndex,
-      renderedLastItemIndex,
-      renderedTotalBefore,
-      renderedTotalAfter,
-      totalRendered
+      totalItems
     } = this;
 
+    let renderedFirstItemIndex, renderedLastItemIndex, renderedTotalBefore, renderedTotalAfter;
+
+    if (renderAll === true) {
```

Comment:
Side-effect of this change is that `renderAll` will supercede `initialRenderCount`, which I think makes sense. The order could be reversed here though, meaning that `initialRenderCount` would always win and `renderAll` would only apply after the Radar has started.